packages/orchestrator-ui-components/src/components/WfoWorkflowSteps/WfoStepList/WfoStepList.tsx

```diff
@@ -69,10 +69,13 @@ export const WfoStepList = React.forwardRef(
         }));
 
         const getReferenceCallbackForStepId =
-            (stepId: string) => (node: HTMLDivElement | null) =>
-                node
-                    ? stepReferences.current.set(stepId, node)
-                    : stepReferences.current.delete(stepId);
+            (stepId: string) => (node: HTMLDivElement | null) => {
```

Comment:
This does not seem related to updating package versions. What are you trying to fix here?

Reply:
It's related to the typescript version upgrade. This fixes the error it throws after the upgrade.